doc/gettingStarted/Applications.md

```diff
@@ -0,0 +1,102 @@
+# Applications
+
+InfiniTime has 13 apps on the `main` branch at the time of writing.
+
+## List of apps
+- Stopwatch
+- Alarm
+- Timer
+- Steps
+- Heartrate
+- Music
+- InfiniPaint
+- Pong
```

Comment:
IIRC, this game is technically called "Paddle" (for copyright reasons?) so we should probably name it accordingly here.
```suggestion
- Paddle
```